detox/package.json

```
@@ -20,7 +20,7 @@
     "babel-plugin-module-resolver": "5.0.0",
     "client-oauth2": "4.3.3",
     "deepmerge": "4.3.1",
-    "detox": "20.11.4",
+    "detox": "^20.19.3",
```

Let's try to keep fixed versions.
```suggestion
    "detox": "20.19.3",
```